src/theme.js

```diff
@@ -0,0 +1,50 @@
+export const lightOrange = "#ffc000";
```

Comment:
What do you think of having a testing file for theme.js?

Reply:
I hadn't thought of that.  It's just exporting constants, so I didn't think it needed any testing.